deb/README.md

```diff
@@ -0,0 +1,20 @@
+# Building your own Docker deb package
+
+`.deb` packages can be built from this directory with the following syntax
+
+```shell
+make ENGINE_DIR=/path/to/engine CLI_DIR=/path/to/cli deb
+```
+
+If you want to specify a specific distro:
+
+```shell
+make ENGINE_DIR=/path/to/engine CLI_DIR=/path/to/cli ubuntu
+```
+
+If you want to specify a specific distro version:
+
+```shell
+make ENGINE_DIR=/path/to/engine CLI_DIR=/path/to/cli ubuntu-xenial
+```
+
```

Comment:
What could be added here is;

- an explanation where the artefacts end up
- perhaps a full example;

```bash
git clone -b <branch-foo> https://github.com/docker/docker-ce.git


make ENGINE_DIR=docker-ce/components/engine CLI_DIR=docker-ce/components/cli ubuntu
```

I'm not sure if we should show how to use github.com/moby/moby, but open to suggestions for that one 👍